src/main/java/uk/gov/digital/ho/hocs/info/domain/entity/EntityResource.java

```diff
@@ -32,7 +32,7 @@ public EntityResource(EntityService entityService) {
     }
 
     @GetMapping(value = "/entity/list/{name}", produces = APPLICATION_JSON_UTF8_VALUE)
-    @Cacheable(value = "getEntitiesForListName", unless = "#result == null || #name == 'MPAM_CAMPAIGNS'", key = "#name")
+    @Cacheable(value = "getEntitiesForListName", unless = "#result == null || #name == 'MPAM_CAMPAIGNS' || #name == 'EXGRATIA_BUS_REPS'", key = "#name")
     public ResponseEntity<List<EntityDto>> getEntitiesForListName(@PathVariable String name) {
```

Comment:
I know this isn't part of your change, but I'd be tempted to chnage to listName to make consistant